src/components/Filters.js

```diff
@@ -26,3 +25,13 @@ export const Filters = ({ styling, filterColumns, onChange, dataSet }) => (
     ))}
   </div>
 );
+
+export const generateFilterOptions = (inputItem, inputDataSet) => {
+  // Receives input data and input item (based on filters) and dynamically generated options to be used for Dropdown component.
+  const uniqueValues = [...new Set(inputDataSet.map(item => item[inputItem]))];
+  return uniqueValues.map(item => ({
```

Comment:
`return` can be avoided. directly `map` on `[...new Set(inputDataSet.map(item => item[inputItem]))]`

Reply:
https://github.com/addania/challenge/pull/21/commits/0302ac5fe529f21eaf3b4a44d6a35d54c9f10c00